src/Tempest/Database/Migrations/MigrationManager.php

```diff
@@ -65,14 +65,14 @@ public function down(): void
             $existingMigrations,
         );
 
-        foreach ($this->databaseConfig->migrations as $migrationClassName) {
+        foreach ($this->databaseConfig->get('desc') as $migrationClassName) {
```

Comment:
when reversing the table creation, we need the migration but in reverse.

Otherwise, if you use foreign keys, you'd get an constraint exception. Because the referenced table got dropped first, but has a reference elsewhere. 💥

Reply:
Circling back to my previous comment, I think I'd prefer to move the array sorting within the migration manager, so that we can get rid of that `get` method with its hard-coded string for asc/desc entirely.